lib/runtime-core/src/backend.rs

```diff
@@ -219,10 +219,11 @@ pub trait RunnableModule: Send + Sync {
     }
 
     /// A wasm trampoline contains the necessary data to dynamically call an exported wasm function.
-    /// Given a particular signature index, we are returned a trampoline that is matched with that
+    /// Given a particular signature index, we are returning a trampoline that is matched with that
```

Comment:
I prefer `returned` to `returning` here;

`, we return a trampoline...` is probably best though.

Reply:
I thought it was an English typo.

Reply:
I believe it's grammatically correct and called the "passive voice", it's often discouraged in English though because it takes more words and ends up being less specific. For example, I generally prefer direct writing like, "This function takes a function signature and returns a corresponding trampoline".